packages/client/tests/functional/_utils/setupTestSuiteEnv.ts

```diff
@@ -18,27 +18,24 @@ export async function setupTestSuiteFiles(suiteMeta: TestSuiteMeta, suiteConfig:
   const suiteFolder = getTestSuiteFolderPath(suiteMeta, suiteConfig)
 
   // we copy the minimum amount of files needed for the test suite
-  await fs.copy(path.join(suiteMeta.testDir, 'prisma'), path.join(suiteFolder, 'prisma'))
-  await copyPreprocessed(
-    path.join(suiteMeta.testDir, suiteMeta.testFileName),
-    path.join(suiteFolder, suiteMeta.testFileName),
-    suiteConfig,
-  )
-  await copyPreprocessed(path.join(suiteMeta.testDir, '_matrix.ts'), path.join(suiteFolder, '_matrix.ts'), suiteConfig)
```

Comment:
We don't this anymore since we also adjust current directory imports to point to original files.